src/components/BalanceGame/index.tsx

```diff
@@ -107,9 +98,11 @@ const BalanceGame = () => {
                   disabled={isOptionLocked}
                 >
                   {currentQuestion.optionA}
-                  <S.PercentText isClicked={clickedOption === "A"}>
-                    {selectedPercentage && `${selectedPercentage.optionA}%`}
-                  </S.PercentText>
+                  {selectedPercentage?.optionA !== undefined && (
+                    <S.PercentText isClicked={clickedOption === "A"}>
```

Comment:
```suggestion
                    <S.PercentText $isClicked={clickedOption === "A"}>
```
styled compoennt에서 prop 사용할 때 지금처럼 사용하시면 HTML의 속성인지 구분이 가지 않아서 콘솔 창에 에러가 생길 것 같은데 `$` 접두사로 HTML 속성이 아니라는 것을 알려주면 좋을 것 같아요~!

참고자료
- https://codinghatso.tistory.com/entry/Transient-props%EB%9E%80-%EB%AC%B4%EC%97%87%EC%9D%B8%EA%B0%80

Reply:
네 수정하고 다시 말씀 드리겠습니다!